gamemode/modules/language/sh_language.lua

```diff
@@ -1,10 +1,18 @@
 local rp_languages = {}
-local selectedLanguage = GetConVar("gmod_language"):GetString() -- Switch language by setting gmod_language to another language
+local selectedLanguage = "en" -- Switch language by setting gmod_language to another language
 
 cvars.AddChangeCallback("gmod_language", function(cv, old, new)
     selectedLanguage = new
 end)
 
+hook.Add("Think", "DarkRPSetLanguage", function()
```

Comment:
Shouldn't this be covered by the `cvars.AddChangeCallback` call above? After all, the moment the car goes from empty to set, that function should trigger.

Maybe the empty string check can be moved there instead. That would remove the need of the separate hook.

Reply:
Because the `cvars.AddChangeCallback` above doesn't work at all no matter what language codes I changed from server-side. This could be the bug directly from Facepunch...

So, I have to create a separate hook to deal with this.

Reply:
Seems `gmod_language` only works for client side by design.

I haven't found a way to make `cvars.AddChangeCallback("gmod_language", ...)` work on server side.